plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -46,7 +46,6 @@ import { PluginPermissionMetadataCollector } from './plugin-endpoints';
 import {
   validateEntityReference,
   validatePolicy,
-  validateQueries,
```

Comment:
Can you remove validateQueries from policies-validation.ts, since it isn't used any more?